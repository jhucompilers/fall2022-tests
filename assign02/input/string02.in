var s;
s = "hello, world";
println(substr(s, 2, 7));
