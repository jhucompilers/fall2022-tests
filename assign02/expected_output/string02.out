llo, wo
Result: 0
